src/main/java/org/example/Main.java

```diff
@@ -4,16 +4,25 @@
 
 public class Main {
 
-  private Scanner scanner;
-  private StudentManager studentManager;
+  private Scanner sc;
 
   public Main() {
-    this.scanner = new Scanner(System.in);
-    this.studentManager = new StudentManager();
+    this.sc = new Scanner(System.in);
   }
 
   public static void main(String[] args) {
+    Main main = new Main();
+    main.InputManager();
+  }
 
-    System.out.println("入力例:");
+  public void InputManager() {
```

Comment:
```suggestion
  public void inputManager() {
```
メソッド名なら小文字始まりで書くのが監修ですね。

Reply:
メソッドは振る舞いなので動詞+目的語のほうがbetter。
```suggestion
  public void manageInput() {
```